flojoy

```diff
@@ -170,7 +170,8 @@ createFlojoyDirectoryWithYmlFile()
       mkdir "$FOLDER" && touch "$FILE" && echo "PATH: $CWD" > "$FILE"
       feedback $? "Created new $FOLDER directory with flojoy.yaml file." 'Failed to create file in the home directory, check the permission or sign in as root user'
    fi
-   CREDENTIALS_FILE=$FOLDER/credentials
+   CREDENTIALS_FILE=$FOLDER/credentials.txt
+   touch $CREDENTIALS_FILE
```

Comment:
Variables in bash should always be wrapped in quotes (e.g `touch "$CREDENTIALS_FILE"`)